src/charm.py

```diff
@@ -510,6 +506,35 @@ def _config_force_https(self) -> None:
         )
         process.wait_output()
 
+    def _on_anonymize_user_action(self, event: ActionEvent) -> None:
+        """Anonymize data from a user.
+
+        Args:
+            event: Event triggering the anonymize_admin_user action.
```

Comment:
is it only for admin users? The function name doesn't indicate that?

Reply:
Description fixed